packages/purpose-process/test/testGetRiskAnalysisConfigurationByVersion.ts

```diff
@@ -0,0 +1,148 @@
+/* eslint-disable @typescript-eslint/no-floating-promises */
+import {
+  getMockEService,
+  getMockTenant,
+  randomArrayItem,
+  writeInReadmodel,
+} from "pagopa-interop-commons-test/index.js";
+import {
+  EServiceId,
+  TenantId,
+  eserviceMode,
+  generateId,
+  tenantKind,
+  toReadModelEService,
+} from "pagopa-interop-models";
+import { describe, expect, it } from "vitest";
+import { genericLogger, getFormRulesByVersion } from "pagopa-interop-commons";
+import {
+  riskAnalysisConfigVersionNotFound,
+  eserviceNotFound,
+  tenantKindNotFound,
+  tenantNotFound,
+} from "../src/model/domain/errors.js";
+import {
+  eservices,
+  purposeService,
+  tenants,
+} from "./purposeService.integration.test.js";
+export const testGetRiskAnalysisConfigurationByVersion = (): ReturnType<
+  typeof describe
+> =>
+  describe("retrieveRiskAnalysisConfigurationByVersion", async () => {
+    it("should retrieve risk analysis configuration by version (Eservice mode: deliver)", async () => {
+      const mockEservice = { ...getMockEService(), mode: eserviceMode.deliver };
+      const kind = randomArrayItem(Object.values(tenantKind));
```

Comment:
I would try to avoid having the random array generated at each run. Either we always test all the cases, or just a specific subset, but without random choices.